tests/sentry/digests/test_utilities.py

```diff
@@ -25,25 +23,50 @@ def sort_records(records):
     return sorted(records, key=lambda r: r.value.event.datetime, reverse=True)
 
 
-class UtilitiesHelpersTestCase(TestCase):
+class UtilitiesHelpersTestCase(TestCase, SnubaTestCase):
+    def create_event(self, project_id):
+        return (
+            self.store_event(
+                data={"timestamp": iso_format(before_now(minutes=1))}, project_id=project_id
+            ),
+        )
+
     def test_get_event_from_groups_in_digest(self):
         project = self.create_project()
         rule = project.rule_set.all()[0]
-        same_group = self.create_group(project=project)
 
         events = [
-            self.create_event(group=same_group),
-            self.create_event(group=same_group),
-            self.create_event(group=self.create_group(project=project)),
-            self.create_event(group=self.create_group(project=project)),
-            self.create_event(group=self.create_group(project=project)),
-            self.create_event(group=self.create_group(project=project)),
+            self.store_event(
+                data={"fingerprint": ["group1"], "timestamp": iso_format(before_now(minutes=1))},
+                project_id=project.id,
+            ),
+            self.store_event(
+                data={"fingerprint": ["group1"], "timestamp": iso_format(before_now(minutes=1))},
+                project_id=project.id,
+            ),
+            self.store_event(
+                data={"fingerprint": ["group2"], "timestamp": iso_format(before_now(minutes=1))},
+                project_id=project.id,
+            ),
+            self.store_event(
+                data={"fingerprint": ["group3"], "timestamp": iso_format(before_now(minutes=1))},
+                project_id=project.id,
+            ),
+            self.store_event(
+                data={"fingerprint": ["group4"], "timestamp": iso_format(before_now(minutes=1))},
+                project_id=project.id,
+            ),
+            self.store_event(
+                data={"fingerprint": ["group5"], "timestamp": iso_format(before_now(minutes=1))},
+                project_id=project.id,
+            ),
         ]
+
         digest = build_digest(
             project, sort_records([event_to_record(event, (rule,)) for event in events])
         )
 
-        events.pop(0)  # remove event with same group
+        events.pop(1)  # remove event with same group
```

Comment:
Why was this working before ?

Reply:
I think it was because of sorting records by timestamp - I changed this back to the original value and gave the events different timestamps